app/src/androidTest/java/org/simple/clinic/facility/FacilitySyncAndroidTest.kt

```diff
@@ -11,13 +11,23 @@ import org.simple.clinic.AppDatabase
 import org.simple.clinic.AuthenticationRule
 import org.simple.clinic.TestClinicApp
 import org.simple.clinic.TestData
+import org.simple.clinic.sync.BaseSyncCoordinatorAndroidTest
 import org.simple.clinic.user.UserSession
 import org.simple.clinic.util.Just
 import org.simple.clinic.util.Optional
 import org.threeten.bp.Instant
 import javax.inject.Inject
 import javax.inject.Named
 
+/**
+ * Facility sync is a special case. We only pull from it, and not push.
+ *
+ * However, [BaseSyncCoordinatorAndroidTest] performs a push operation even in the test for the pull
+ * operation. We can't extend that test over here since facilities aren't supposed to be pushed from
+ * the app at all.
+ *
+ * TODO: Revisit this when making a composite sync android test.
+ **/
```

Comment:
This was another reason why `SyncCoordinator` (previously `DataSync`) was written in a composite manner.

Reply:
Resolving since it seems to be a duplicate comment.